spec/02-integration/02-cmd/02-start_stop_spec.lua

```diff
@@ -18,6 +18,60 @@ describe("kong start/stop #" .. strategy, function()
     helpers.clean_prefix()
   end)
 
+  it("with #referenced values [fails]", function()
+    local _, stderr, stdout = assert(helpers.kong_exec("start", {
+      prefix = helpers.test_conf.prefix,
+      database = helpers.test_conf.database,
+      pg_password = "{vault://env/pg_password}",
+      pg_database = helpers.test_conf.pg_database,
+      cassandra_keyspace = helpers.test_conf.cassandra_keyspace
+    }))
+    if strategy == "postgres" then
+      assert.matches("referenced value has a wrong type, expected 'string', got 'nil'", stderr, nil, true)
+    end
+    assert.matches("Kong started", stdout, nil, true)
```

Comment:
I think Kong should not start if the secret isn't present. Seems like a possible security problem.

Reply:
Yes, we are still doing this "how to manage errors" on each case.